.github/workflows/tests.yml

```diff
@@ -50,6 +103,8 @@ jobs:
         coverage run --append -m scalems.local --log-level=info examples/basic/echo.py hi there
         coverage xml -o coverage-cli.xml
         python -c 'if open("0000000000000000000000000000000000000000000000000000000000000000/stdout", "r").readline().rstrip() != "hi there": assert False'
+        (cd examples/lammps_files/simple_lammps && python lammps_from_python.py)
+        (cd examples/lammps_files/simple_lammps && mpirun -np 2 python lammps_from_python_parallel.py)
```

Comment:
The output for these (and for the LAMMPS build) can be viewed through "Checks": https://github.com/SCALE-MS/scale-ms/pull/158/checks?check_run_id=3321947946

It seems right to me.

In a follow-up change, we can start some basic testing that RP dispatching works right and can activate the environment correctly.

Reply:
I assume this is at least good enough for now. If @mrshirts or @Jianming-C find additional changes necessary, we can address in a follow-up change.